client/src/components/TrailShowPage.js

```diff
@@ -23,6 +25,17 @@ const TrailShowPage = (props) => {
     getTrail()
   }, [])
 
+  const getReviews = trail.reviews.map((review) => {
+    return (
+      <div>
+        <ul>
+          <li>Comment: {review.comment}</li>
+          <li>Rating: {review.rating}</li>
+        </ul>
+      </div>
```

Comment:
The `ul` can remain as the top-level element here since it wraps around both `li` tags. Including the `div` is actually repetitive if you look where `getReviews` is returned down below, it's also wrapped in a `div`